x/ibc/core/04-channel/types/errors.go

```diff
@@ -25,4 +25,7 @@ var (
 	ErrPacketReceived            = sdkerrors.Register(SubModuleName, 18, "packet already received")
 	ErrAcknowledgementExists     = sdkerrors.Register(SubModuleName, 19, "acknowledgement for packet already exists")
 	ErrInvalidChannelIdentifier  = sdkerrors.Register(SubModuleName, 20, "invalid channel identifier")
+
+	// Antehandler error
+	ErrRedundantTx = sdkerrors.Register(SubModuleName, 21, "packet messages are redundant")
```

Comment:
```suggestion
	ErrRedundantTx = sdkerrors.Register(SubModuleName, 22, "packet messages are redundant")
```
This might be less confusing for relayers connected to chains with different ibc-go versions

Reply:
ok, thanks. I was not sure if I should leave `22` or just keep consecutive number.